src/warplib/generateWarplib.ts

```diff
@@ -43,15 +39,6 @@ export const warplibFunctions: WarplibFunctionInfo[] = [
   shl(),
   shr(),
   shr_signed(),
-  // ge - handwritten
-  ge_signed(),
```

Comment:
Is it save to remove operations for signed integers as well? I thought those were not supported yet.